apache/Dockerfile

```diff
@@ -28,6 +28,12 @@ RUN curl -o mautic.zip -SL https://s3.amazonaws.com/mautic/releases/${MAUTIC_VER
 COPY docker-entrypoint.sh /entrypoint.sh
 COPY makeconfig.php /makeconfig.php
 COPY makedb.php /makedb.php
+RUN tar xf /usr/src/php.tar.xz \
+        -C /usr/local/etc/php/ \
+        --strip-components 1 \
+        php-5.6.23/php.ini-production \
```

Comment:
Would it be more straightforward to include the php.ini to the mautic repo with `always_populate_raw_post_data` enabled and ADD it from the Dockerfile?

Alternatively, tar supports wildcards which would avoid the version-specific path, like so:
```
RUN tar xf /usr/src/php.tar.xz \
        -C /user/local/etc/php/ \
        --strip-components 1 \
        --wildcards php-*/php.ini-production \
```